x-pack/plugins/actions/README.md

```diff
@@ -81,13 +81,13 @@ Table of Contents
 **Action Type**: A programatically defined integration with another service, with an expected set of configuration and parameters properties, typically defined with a schema. Plugins can add new
 action types.
 
-**Action**: A configuration object associated with an action type, that is ready to be executed. The configuration is persisted via Saved Objects, and some/none/all of the configuration properties can be stored encrypted.
+**Action**: A configuration object associated with an action type, that is ready to run. The configuration is persisted via Saved Objects, and some/none/all of the configuration properties can be stored encrypted.
 
 ## Usage
 
 1. Develop and register an action type (see [Action types -> Example](#example)).
 2. Create an action by using the [RESTful API](#restful-api).
-3. Use alerts to execute actions or execute manually (see [Firing actions](#firing-actions)).
+3. Use alerts to generate actions or create them manually (see [Firing actions](#firing-actions)).
```

Comment:
create seems wrong as actions aren't created.
They're either attached to a rule which will then schedule them when an alert is created, or they're triggered manually.

```suggestion
3. Use alerting rules to generate actions or trigger them manually (see [Firing actions](#firing-actions)).
```